docs/resources/security_certificate.md

```diff
@@ -0,0 +1,163 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "netapp-ontap_security_certificate Resource - terraform-provider-netapp-ontap"
+subcategory: ""
+description: |-
+  SecurityCertificate resource
+---
+
+# netapp-ontap_security_certificate (Resource)
+
+Create/ install/ sign a certificate
+
+### Related ONTAP commands
+```commandline
+* security certificate show
+* security certificate create
+* security certificate install
+* security certificate sign
+```
+
+## Supported Platforms
+* On-prem ONTAP system 9.6 or higher
```

Comment:
Just want to make sure if you check the fsx is supported or not.

Reply:
No, @chuyich , I haven't tested for fsx support.
Could you please let me know the steps that are required to test the same?

Reply:
Please check the slack message.

Reply:
@csahu9 @wenjun666 has confirmed the fsx is supported.
Please add the following line:
* Amazon FSx for NetApp ONTAP